src/notification/Notification.js

```diff
@@ -0,0 +1,66 @@
+import GrouperClient from "../structures/GrouperClient";
+
+export default class Notification {
+    /**
+     * Creates a new store for commands
+     * 
+     * @param {GrouperClient} client 
+     */
+    constructor(client, guild, channel) {
+        /**
+         * GrouperClient
+         * 
+         * @type {GrouperClient}
+         */
+        this.client = client;
+        let supportGuild = null;
+        client.guilds.forEach((g) => {
+            if (g.id == guild) {
+                supportGuild = g;
+                return;
+            }
+        });
+
+        if (!supportGuild)
+            throw new Error("Unable to find support guild, has this been entered incorrectly?");
+
+        let supportChannel = null;
```

Comment:
Perhaps `let supportChannel = supportGuild.channels.find(c => c.id == channel);` as opposed to the forEach. Again you'll want to test this snippet before you implement it though.